content/tools/ruby.md

```diff
@@ -112,10 +112,32 @@ rescue LoadError
 end
 ```
 
-## Ruby-on-Rails
+## Ruby on Rails
 
-### Rails 3.x and above
+### cucumber-rails
 
-Go to Cucumber-Rails' [README](https://github.com/cucumber/cucumber-rails/blob/master/README.md) for installation instructions.
+[cucumber-rails](https://github.com/cucumber/cucumber-rails) is a RubyGem which
+brings Ruby on Rails Generators for Cucumber with special support for Capybara
+and DatabaseCleaner.
 
-As of Cucumber-Rails version 0.5.0 and above, Cucumber-Rails **only** supports Rails 3.x and above (Rails 2 is **not** supported).
\ No newline at end of file
+#### Installing
+
+The `cucumber:install` generator sets up Cucumber in your Rails project. It
```

Comment:
`cucumber:install` command

Reply:
Oh, it's called a Rails Generator, is it not? https://github.com/cucumber/cucumber-rails#installation

Reply:
Meh oki. I always think of generators as the rails ones to generate apps. Leave this as is then :)